### PR TITLE
Numeric Animation

### PR DESCRIPTION
**Pull request by m-ata**

### CODE REVIEW

File: src/components/NumericAnimation/NumericAnimation.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+              width: '100%',
+              height: 120,
+              overflow: 'auto',
+              overflowY: 'hidden',
+              fontSize: '120px',
+            }}
+            scrollTop={
+              index === numberArray.length - 1 ? transitionFast.scroll : transitionSlow.scroll
+            }
+          >
```
Comment (miedziak) [inline src/components/NumericAnimation/NumericAnimation.tsx:85]: todo: same here, we can use styled components here, you can probably do something like 
```js
const AnimatedContainer = styled(animated.div)`
...
`
```

**Review by miedziak — CHANGES_REQUESTED** [review]
GJ, just some minor things

File: src/components/NumericAnimation/NumericAnimation.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  return (
+    <Box style={{ display: 'flex' }}>
+      {numberArray.map((num, index) => {
+        if (Number.isNaN(num)) {
+          return (
+            <Box
+              style={{
+                height: 120,
+                fontSize: '120px',
+              }}
```
[the remark this answers is not in the document]
Comment (m-ata) [reply src/components/NumericAnimation/NumericAnimation.tsx:63]: done!

File: src/components/NumericAnimation/NumericAnimation.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          <animated.div
+            key={`${index + 1}`}
+            ref={ref}
+            style={{
+              position: 'relative',
+              width: '100%',
+              height: 120,
+              overflow: 'auto',
+              overflowY: 'hidden',
+              fontSize: '120px',
```
[the remark this answers is not in the document]
Comment (m-ata) [reply src/components/NumericAnimation/NumericAnimation.tsx:80]: done!

File: src/components/NumericAnimation/NumericAnimation.tsx (hunk 1 above)
Comment (m-ata) [reply src/components/NumericAnimation/NumericAnimation.tsx]: Good suggestion. Implemented now.

**Review by miedziak — APPROVED** [review]

**Pull request merged**